react/react_intro/README.md

```diff
@@ -130,23 +142,27 @@ function App() {
 export default App;
 ```
 
-`App` is a function with an odd-looking return value.  Inside the `()`, we see code that looks like a mixture of JavaScript and HTML.  This is a special syntax for React called `JSX`.  It is an extension to JavaScript that makes it easy to build React elements.  The following line is perfectly valid in JSX:
+`App` is a function with an odd-looking return value. Inside the `()`, we see code that looks like a mixture of JavaScript and HTML. This is a special syntax for React called **JSX**.
 
-```js
+While this might be jarring at first, JSX alleviates a big problem we've been facing up to this point. Working with the DOM, we had a JS file referring to, and updating, elements in an HTML file. However, moving back and forth between these files is clunky and unnatural. We'd much rather have our elements and their functionality in the same place. That's what JSX syntax offers us.
```

Comment:
Really like this explanation here and I think it could be highlighted in the earlier paragraph about why React was created / is worth learning. As a nitpick, I might replace `clunky and unnatural` with something more concrete and less opinionated. For example, `moving back and forth between these files is labor intensive, hard to maintain, and mistake-prone.`